src/System.Reflection.MetadataLoadContext/tests/System.Reflection.MetadataLoadContext.Tests.csproj

```diff
@@ -0,0 +1,74 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <ProjectGuid>{443B7659-B3FA-4B32-88BA-3A0517E21018}</ProjectGuid>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Configurations>netcoreapp-Debug;netcoreapp-Release;netstandard-Debug;netstandard-Release</Configurations>
+    <LangVersion>7.2</LangVersion>
```

Comment:
This shouldn't be necessary. We already point to latest: https://github.com/dotnet/corefx/blob/24db8490eb6bdeeb2c4353d5b7f9b786546cfa2b/Directory.Build.props#L315

Reply:
It was necessary at the time it was added when building+running the tests from VS since it didn't target the latest yet. However, it is now no longer necessary so I removed it.